### DISCUSSION TITLE
Record plugin cannot be added in React project

### DISCUSSION THREAD

**Discussion by hattonjs** [katspaugh/wavesurfer.js#3419] [Q&A]
Record plugin cannot be added in React project.  Followed the instructions implicitly.  Any solutions to this?

**Comment by katspaugh** [comment]
There's not enough info to be able to help you. Most likely you're importing it wrong.

**Reply by hattonjs** [reply]
Maybe you can enlighten me as to the import for React then … because I have followed all the instructions completely.  Just a standard npm install.

**Reply by katspaugh** [reply]
No, sorry, I can't help with this attitude.

**Reply by hattonjs** [reply]
What attitude?

**Reply by katspaugh** [reply]
I said already that I lack information. You created a bug report completely dismissing the template for it that contains required sections like a minimal code sample, browser etc.
Ultimately, you'll need to debug your code and I can't help you with that w/o even seeing what you tried.
